### PR TITLE
Step that can be used to download resources from AWS S3

### PR DESCRIPTION
**Pull request by igorcferreira**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/s3-download/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+source_code_url: https://github.com/FutureWorkshops/bitrise-step-s3-download
+published_at: 2020-10-13T13:41:06.598528+02:00
+source:
+  git: https://github.com/FutureWorkshops/bitrise-step-s3-download.git
+  commit: e8db70d94d97e74dfebf9e916f38020f89dc83ad
+host_os_tags:
+- osx-10.10
+- osx-10.9
+project_type_tags:
+- ios
```
Comment (koral--) [inline steps/s3-download/1.0.0/step.yml:15]: This step should not be restricted to particular kind of projects.
No project type tags should be specified.

File: steps/s3-download/1.0.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+- android
+- xamarin
+type_tags:
+- amazon
```
Comment (koral--) [inline steps/s3-download/1.0.0/step.yml:19]: There are no such types: https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#step-grouping-convention

File: steps/s3-download/1.0.0/step.yml (hunk 1 above)
Comment (igorcferreira) [reply steps/s3-download/1.0.0/step.yml:15]: 👍

File: steps/s3-download/1.0.0/step.yml (hunk 2 above)
Comment (igorcferreira) [reply steps/s3-download/1.0.0/step.yml:19]: 👍

**Comment by igorcferreira** [comment]
I'll close this PR and open a new one with the corrections properly made, after updating the Step definition.

**Pull request closed without merging**